spec/operations/rating_books/create_spec.rb

```diff
@@ -1,34 +1,37 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'faker'
 
-RSpec.describe ::RatingBooks::Create do
-  context 'validation tests' do
-    before(:each) do
-      @user = create(:user)
-      @book = create(:book)
+describe ::RatingBooks::Create do
+  let(:book) { create(:book).decorate }
+  let(:user) { create(:user) }
+  let(:rating) { create(:rating) }
+  let(:other_rating) { create(:rating) }
+  subject(:create_rate) { ::RatingBooks::Create.call(user:, book:, rating:) }
+  subject(:create_other_rate) { ::RatingBooks::Create.call(user:, book:, rating: other_rating) }
+
+  context 'check if user' do
+    it 'wasnt rated book' do
+      expect(book.rated_by?(user)).to eql(false)
     end
 
-    it 'not rated before' do
-      rating = create(:rating)
-      expect(@book.decorate.rated_by?(@user)).to eql(false)
-      ::RatingBooks::Create.call(user: @user, book: @book.decorate, rating:)
-      expect(@book.decorate.rated_by?(@user)).to eql(true)
-      expect(@user.rating_books.where(book_id: @book.id, rating_id: rating.id)).to exist
+    it 'was rated book' do
+      create_rate
+      expect(book.rated_by?(user)).to eql(true)
```

Comment:
there is no need to check this.
your test goal is to check if rating_books is created and to check if the rating is changed when the user's already rated the book.